bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -735,7 +735,7 @@ func (r *KubeadmConfigReconciler) reconcileDiscovery(ctx context.Context, cluste
 		}
 
 		config.Spec.JoinConfiguration.Discovery.BootstrapToken.Token = token
-		log.Info("Altering JoinConfiguration.Discovery.BootstrapToken", "Token", token)
+		log.Info("Altering JoinConfiguration.Discovery.BootstrapToken", "Token")
```

Comment:
The arguments to `Info` are supposed to be the message followed by key/value pairs, so just having `"Token"` isn't right.
```suggestion
		log.Info("Altering JoinConfiguration.Discovery.BootstrapToken")
```

Reply:
Thanks, apologies for not reading the logr docs thoroughly.